manifests/init.pp

```diff
@@ -17,6 +19,20 @@
     purge   => $config_dir_purge,
   }
 
+  if str2bool($cron_purge) {
```

Comment:
I'm not following... `validate_bool()` is in place as a guard function so `str2bool()` could only be operating on a bool.